src/CloudFoundry.VisualStudio/MSBuild/MSBuildProcess.cs

```diff
@@ -0,0 +1,105 @@
+﻿namespace CloudFoundry.VisualStudio.MSBuild
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Text;
+    using System.Threading.Tasks;
+    using Microsoft.Build.Evaluation;
+    using EnvDTE;
+    using Microsoft.VisualStudio.Shell;
+ 
+    public class MSBuildProcess : IMSBuildProcess
+    {
+        public void Push(string projectFullPath, string pubxml, string logFile, Microsoft.Build.Framework.LoggerVerbosity verbosity)
```

Comment:
Can we call this method something else (like `Publish`)? Is it restricted to Push, or does it execute an msbuild publish?